### PR TITLE
[Components with more than 1 service] LPS-206064 Refactor OAuth2AuthorizationExpandoPortalInstanceLifecycleListener component

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/constants/SAPEntryObjectConstants.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+package com.liferay.osb.faro.web.internal.constants;
+
+import com.liferay.osb.faro.web.internal.application.ApiApplication;
+import com.liferay.osb.faro.web.internal.controller.api.RecommendationController;
+import com.liferay.osb.faro.web.internal.controller.api.ReportController;
+
+/**
+ * @author Joao Victor Alves
+ */
+public class SAPEntryObjectConstants {
```
Comment (rafaprax) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/java/com/liferay/osb/faro/web/internal/constants/SAPEntryObjectConstants.java:15]: rename to `FaroSAPConstants`

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MrJohn1911** [comment]
ci:test:sf

**Comment by rafaprax** [comment]
forwarded to https://github.com/liferay-core-infra/liferay-portal/pull/6459

**Review by rafaprax — APPROVED** [review]

**Pull request closed without merging**